crates/polars-plan/src/dsl/mod.rs

```diff
@@ -1308,6 +1308,21 @@ impl Expr {
         self.apply_private(FunctionExpr::Rank { options, seed })
     }
 
+    #[cfg(feature = "replace")]
+    /// Replace the given values with other values.
+    pub fn replace<E: Into<Expr>>(self, old: E, new: E, default: Option<E>) -> Expr {
+        match default {
+            Some(default) => {
+                let arguments: &[Expr] = &[old.into(), new.into(), default.into()];
+                self.apply_many_private(FunctionExpr::ReplaceWithDefault, arguments, false, false)
```

Comment:
Can we create a single `FunctionExpr::Replace` with a `default` boolean flag?

This saves some compiler bloat, as the derive macros, `clone, debug, serialize` etc. create code for every variant.

Reply:
Done!